tools/ci/array-api-xfails.txt

```diff
@@ -7,10 +7,14 @@ array_api_tests/test_signatures.py::test_func_signature[reshape]
 # 'min/max' args are present. 'a_min/a_max' are retained for backward compat.
 array_api_tests/test_signatures.py::test_func_signature[clip]
 
-# missing 'descending' keyword arguments
+# missing 'descending' keyword argument
 array_api_tests/test_signatures.py::test_func_signature[argsort]
 array_api_tests/test_signatures.py::test_func_signature[sort]
 
+# missing 'descending' keyword argument
+array_api_tests/test_sorting_functions.py::test_argsort
+array_api_tests/test_sorting_functions.py::test_sort
```

Comment:
The rest of these are bugs in the test suite but this looks like one thing that is actually missing from NumPy for array API compatibility.

Reply:
I agree, but we decided that it would be too much churn to include it now.